### PR TITLE
mobile to desktop version

### PR DESCRIPTION
**Pull request by PowerLevel9000**
making responsive to desktop version and mobile version in future there will be a tablet version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -408,3 +413,194 @@ footer textarea {
...
+
+  .info2 {
+    width: 70%;
+  }
+
+  .experince {
+    width: 45%;
+    margin-top: 163px;
+    margin-right: auto;
+  }
```
Comment (emmiiorji) [inline styles.css]: Kindly modify the languages section to be similar to the figma design.

Note the following:

- The arrow on figma design is lighter than on your design
- The icons has more passing on the figma design than your design
- You're missing the horizontal lines
- There's more margin on the figma design between the arrows and the container than on your design

Your design:
image


Figma design:
image

File: styles.css [hunk 2]
```diff
@@ -408,3 +413,194 @@ footer textarea {
...
+    width: 80%;
+    height: auto;
+    flex-direction: row;
+    margin-bottom: 142px;
+  }
+
+  .project div {
+    margin-top: -68px;
+    margin-left: 32px;
+  }
```
Comment (emmiiorji) [inline styles.css:537]: Please, modify the project cards to match the design as closely as possible. Kindly ensure that the description takes up the correct width as shown on the design. Also the list of technologies are left aligned.

Your design:
image


Figma design:
image

File: styles.css [hunk 3]
```diff
@@ -408,3 +413,194 @@ footer textarea {
   padding-left: 10px;
   padding-top: 10px;
 }
+
+@media screen and (min-width: 376px) {
```
Comment (emmiiorji) [inline styles.css]: Kindly ensure that the mobile version remains responsive until the desktop screen size (768px). This means that the value for the min-width property should be 768px or above. This screen sizes below are supposed to display as mobile.

Your design below 768px:
image

Your design at 515px:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @PowerLevel9000, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @PowerLevel9000 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingno3l** [comment]
Congratulations!!!

**Comment by PowerLevel9000** [comment]
# Thank you 💪💪💪🙏🙏🙏

ThankYouSoMuchPrabhasGIF


Dear @emmiiorji @binhussen ,

I wanted to express my gratitude for your willingness to take the time to review my code on GitHub. Your feedback and suggestions have been instrumental in improving the quality of my work, and I appreciate your insights and expertise in this area.

It's inspiring to have someone like you, who is knowledgeable and experienced, provide valuable feedback that helps me enhance my coding skills. I truly appreciate the time and effort you put into reviewing my code, and I'm grateful for your constructive criticism.

If there are any areas that I need to work on, please don't hesitate to let me know. I'm open to hearing any suggestions or recommendations you may have. Your feedback is essential to my growth as a developer, and I value it tremendously.

Thank you again for your time and for sharing your expertise with me. I'm looking forward to applying the insights gained from your review to my future projects.

Sincerely,
@PowerLevel9000

**Pull request merged**